WordPress/twentytwentyone
# Fix alignement issue on alignleft pullquote. Fixes #359.

Fixes #359.

## Summary
Fix alignement issue on alignleft pullquote.

## Relevant technical choices:
No.

## Test instructions

This PR can be tested by following these steps:

1. Add a pull quote to a post or 
2. Add a text quote and citation to the 
3. See the position of the 
4. Save and view the citation on the front.


## Quality assurance
